android/TPSDropDown/app/src/main/java/com/gettipsi/tpsdropdown/Adapter.java

```diff
@@ -0,0 +1,39 @@
+package com.gettipsi.tpsdropdown;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import java.util.List;
+
+public class Adapter extends ArrayAdapter<Object> {
+
+    public Adapter(Context context, int resource, List<Object> objects) {
+        super(context, resource, objects);
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        ViewHolder holder;
+        if (convertView == null) {
+            convertView = LayoutInflater.from(getContext()).inflate(android.R.layout.simple_list_item_1, parent, false);
+            holder = new ViewHolder(convertView);
+            convertView.setTag(holder);
+        } else {
+            holder = (ViewHolder) convertView.getTag();
+        }
+        holder.text.setText(getItem(position).toString());
+        return convertView;
+    }
+
+    private class ViewHolder {
+        TextView text;
+
+        public ViewHolder(View view) {
+            text = (TextView) view.findViewById(android.R.id.text1);
```

Comment:
This default android id and default android layout, after applying styles it will be changed